cmd/openshift/kodata/tekton-addon/0.0.1/optional/quickstarts/install-associate-pipeline-quickstart.yaml

```diff
@@ -57,7 +57,7 @@ spec:
     description: |-
       ### You’ve just explored the topology of your application and seen it’s related resources.  Now let’s start your pipeline.
       Notice the Pipeline Runs section of the Side Panel
-      1. The first row shows the Pipeline associated with the application and an action button to **Start Last Run**, which is disabled.
+      1. The first row shows the Pipeline associated with the application and an action button to **Start Last Run**.
       2. Click on the Pipeline link, which should bring you to the **Pipeline Details** page.
       3. From the Action menu, click on **Start** to start your Pipeline.
```

Comment:
The pipeline is already running so there is no value in starting it again, what about changing this step to something like "sit and watch the pipeline"

Reply:
Navigating to pipeline's Details page to see more details would be useful

Reply:
+1 to navigating to details page but not starting it second time.

Reply:
@karthikjeeyar @ppitonak is this resolved.
shall we merge this

Reply:
We can go ahead with this PR since I made a minimal changes here, but in order to have a completely new content for the 4.7 pipeline flow, we could have a separate ticket for that. WDYT  @ppitonak @nikhil-thomas?